docs/sources/getting-started/_index.md

```diff
@@ -7,6 +7,29 @@ weight: 9
 
 # Get started
 
-This section provides guidance on how build your first dashboard after you have installed Grafana. It also provides step-by-step instructions on how to add a Prometheus, InfluxDB, or an MS SQL Server data source. Refer to [Data sources]({{< relref "../administration/data-source-management/" >}}) for a list of all supported data sources.
+Getting started with Grafana involves three steps:
+
+1. Setup Grafana
+2. Connect your first data source
```

Comment:
- It's preferable to use Markdown's auto-incrementing ordered lists. 
  To use the auto-incrementing lists, always use the prefix `1.`.

```suggestion
1. Connect your first data source
```